packages/browser-sdk/src/lib/react/Grid/VideoCell/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+
+import { fitToBounds } from "../layout/gridUtils";
+
+interface Props {
+    aspectRatio: number;
+    children: React.ReactElement;
+    className?: string;
+    height: number;
```

Comment:
Can we simply provide these properties in a CSS class and apply that to the target object here?

Developers can further customize these elements by passing in their own `className`. If we wanted to go further we could give the target elements id attributes and developers can simply write a CSS file that overloads those elements in the DOM?